polkadot/xcm/pallet-xcm/src/precompiles/IXcm.sol

```diff
@@ -1,29 +1,45 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-/// @title Defines all functions that can be used to interact with XCM
-/// @dev Parameters MUST use SCALE codec serialisation
+/// @dev The XCM Precompile contract address.
+address constant XCM_PRECOMPILE_ADDRESS = address(0xA0000);
+
+/// @title An interface for interacting with `pallet_xcm`
+/// @notice Provides functions for executing and sending XCM messages.
```

Comment:
Can you point out that this is a low level interface that essentially just forwards to the dispatchable?